lua/nvim-treesitter-playground/printer.lua

```diff
@@ -3,43 +3,45 @@ local api = vim.api
 
 local M = {}
 
-local function print_tree(root, results, indent)
+local function print_tree(root, unnamed, results, indent)
   local results = results or { lines = {}, nodes = {} }
   local indentation = indent or ""
 
   for node, field in root:iter_children() do
-    if node:named() then
+    if node:named() or unnamed then
+      local name = node:type()
+      if not node:named() then
+        name = '"' .. name .. '"'
+        -- quote any possible newlines (newlines cause errors in nvim_buf_set_lines)
+        name = name:gsub('\n', '\\n')
+      end
+
       local line
       if field then
         line = string.format("%s%s: %s [%d, %d] - [%d, %d]",
-          indentation,
-          field,
-          node:type(),
-          node:range())
+          indentation, field, name, node:range())
       else
         line = string.format("%s%s [%d, %d] - [%d, %d]",
-          indentation,
-          node:type(),
-          node:range())
+          indentation, name, node:range())
       end
 
       table.insert(results.lines, line)
       table.insert(results.nodes, node)
 
-      print_tree(node, results, indentation .. "  ")
+      print_tree(node, unnamed, results, indentation .. "  ")
     end
   end
 
   return results
 end
 
-function M.print(bufnr, lang)
+function M.print(bufnr, unnamed, lang)
```

Comment:
Move this to the end of the arguments list since lang would be the more common passed in argument. This also avoids an uneeded breaking change in the public API.